packages/connect-web/src/index.ts

```diff
@@ -42,71 +42,71 @@ import {
 
 export {
   /**
-   * @deprecated please import from @bufbuild/connect instead
+   * @deprecated please import from \@bufbuild/connect instead
```

Comment:
just to double check: this definitely is great as a fix for the escaping, but you know it's still the case that users will not be able to see this message, right?  are you planning to tackle that in a separate PR?

Reply:
I'm not sure there is a good solution to this. See https://github.com/microsoft/TypeScript/issues/42684 and https://github.com/microsoft/TypeScript/pull/47293